### DISCUSSION TITLE
Major improvements in v5.7

### DISCUSSION THREAD

**Discussion by gileli121** [WindowTop/WindowTop-App#189] [Show and tell]
# Major improvements in v5.7

## Anchors - New style configuration UI with real-time preview (#181)
![anchor_settings_demo  ](https://user-images.githubusercontent.com/17680514/148612345-13182b75-2b4a-49de-ba0f-44354ebaf2dd.gif)

## Anchors - Show only window icon (#181)
![anchors_demo](https://user-images.githubusercontent.com/17680514/148513474-4ddbeee7-3a34-4c6d-96cb-e2aae0934688.gif)

## Anchors - Show window icon with title
Now the anchors will show the window icon by default (#177)!
![anchors_demo](https://user-images.githubusercontent.com/17680514/147419085-c0f9f272-f593-44d4-af26-a0f9063119fb.gif)

## Anchors - Dynamic anchor size based on how recently the window used
![image](https://user-images.githubusercontent.com/17680514/147989745-55a313cc-190c-43b6-95f9-0c61d52d8490.png)

## Drag item to window via the anchor (#180)
Now it is possible to drag item to the window directly from the anchor!
![drag_to_anchor_demo](https://user-images.githubusercontent.com/17680514/147726013-d712ce33-0858-465b-aaaf-0a786afebab0.gif)

## Added auto & self-update to the Microsoft Store version
Now the app will be able to update itself without any extra effort! 
You just need to click on the button to update and anything else is done for you!

## Added option to configure the frame border for top-most window (#169)
The set-top feature have option to highlight top-most window with red border/frame. 
previously you could not configure the border width and color.
Now it is possible!
![set_top_preview_demo](https://user-images.githubusercontent.com/17680514/147696937-fb742ddf-6c2d-4da4-a265-dca29a6465d4.gif)

# Disable on Game Mode
Added option to disable the app when game-mode detected
![image](https://user-images.githubusercontent.com/17680514/147696567-41d22fb3-8c0a-4e69-9a61-771acd5dad25.png)

## Added Wizard window
When you first use the program, it will show you a step-by-step wizard that helps you to configure the program for your needs:
The wizard will give you a visual indication of what each option does:
![quick_start_wizard_demo_cmp](https://user-images.githubusercontent.com/17680514/144728898-02b8d299-bc07-487c-835c-1a092cea7e19.gif)

## Getting Started in Settings improved
Now all of these quick start screens that shows on the wizard will be available also from the Home menu in the Settings:
![quick_start_settings_demo_cmp](https://user-images.githubusercontent.com/17680514/144728685-f01b86e4-aa37-4f65-8f3b-71a4de2c7af9.gif)

## Red frame for the top-most window is now available for free
Now the option to highlight the top-most window with a red frame is unlocked also in the free version (#169)

## 30 days trial will start on the first run
Before the change, WindowTop asked you if you want to start 30 days trial.
Now it will not ask. It will just start the 30 days trial on the first run.
After the 30 days trial, it will run with limited functionality and ask you if you want to purchase it

**Comment by PureItTech** [comment]
Trial does not work; it made me purchase the full product to get pro features to work.

**Reply by gileli121** [reply]
It is probably not the first case.
Your case may be the same as 
https://github.com/BiGilSoft/WindowTop/issues/244
The reporter probably preferred to close the ticket because he probably did not want it to be implemented.

If you have any thoughts about it, or you think that what I suggested in #244 is a better idea, I will happy to hear your opinion.

Thank you.
